ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/inventory/group_vars/DC1_POD1.yml

```diff
@@ -8,30 +8,33 @@ spine:
     platform: vEOS-LAB
     bgp_as: "65110.100"
     loopback_ipv4_pool: 172.16.110.0/24
-    uplink_interfaces: ['Ethernet1', 'Ethernet2']
-    uplink_switches: ['DC1-SUPER-SPINE1', 'DC1-SUPER-SPINE2']
+    uplink_interfaces: ["Ethernet1", "Ethernet2"]
```

Comment:
All these other changes only makes it harder to review. It doesn't really matter if we use single or double quotes or mix them. This is just for molecule. Same regarding spacing inside lists.

Reply:
same as above, that is not a manual change